ISICbalance/ViewControllers/AccountViewController.swift

```diff
@@ -14,8 +14,12 @@ import ACKReactiveExtensions
 import ACKategories
 import SVProgressHUD
 
-final class AccountViewController: BaseViewController, ValidateErrorPresentable {
-    private let viewModel: AccountViewModel
+protocol AccountFlowDelegate: class {
+    func balanceActionCompleted(in viewController: AccountViewController)
+}
+
+class AccountViewController: BaseViewController, ValidateErrorPresentable {
```

Comment:
možná bys mohl klidně nasadit nový `BaseViewController` s generickým ViewModelem. Pouze pokud se ti chce, za mě to není nutné